app/assets/stylesheets/lines.scss

```diff
@@ -1,3 +1,21 @@
 // Place all the styles related to the lines controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+input[type="radio"] {
+    -ms-transform: scale(2); /* IE 9 */
+    -webkit-transform: scale(2); /* Chrome, Safari, Opera */
+    transform: scale(2);
+}
+
+label {
+    margin: 10px;
```

Comment:
Absolutely positively do not spend more than 5 minutes on this, but it would rule if we could crank up the text size for labels here. Maybe 16px? Again, if this winds up being a pain, let's punt to later.

Reply:
I bumped up the fontsize to 24px (it was allready at the default 17px) feel free to tweak it up or down
<img width="731" alt="screen shot 2017-09-09 at 1 45 22 pm" src="https://user-images.githubusercontent.com/4335814/30242478-3a99e2f6-9565-11e7-83c8-1d1b8633a2e7.png">

Reply:
<3